src/electron/worker.js

```diff
@@ -144,11 +145,10 @@ class BitgoSingleton {
     ccUtxos.forEach(utxo => {
       if (utxo?.tokendata?.tokenid) {
         res[utxo.tokendata.tokenid.reverse().toString('hex')] = utxo.satoshis;
+      } else if (!!utxo?.tokendata?.name && utxo?.tokendata?.funcid === 'c') {
+        // In case token was created by the wallet, but has no transactions
+        res[utxo.txid.reverse().toString('hex')] = utxo.satoshis;
       }
-      // for testing in case tokens are in the old format
-      // if (utxo?.tokeldata?.tokenid) {
-      //   res[utxo.tokendata.tokenid.reverse().toString('hex')] = utxo.satoshis;
-      // }
     });
     console.groupEnd();
```

Comment:
can prolly get rid of this one and the one above too. They are not consoling anything :D